src/components/MultiSelect/styles.ts

```diff
@@ -99,6 +109,15 @@ export default styled.article`
     }
   }
 
+  .down {
+    position: absolute;
```

Comment:
Make it relative. Set space-between to parent or make it margin-left auto

Reply:
Done